build_tools/ci/run_matmul_test.sh

```diff
@@ -255,7 +264,8 @@ function run_matmul_test() {
   # Make a guess as to whether we need to sign the XCLBIN:
   SIGNER=${XRT_DIR}/amdxdna/setup_xclbin_firmware.sh
   # 1) check if $XRT_DIR/amdxdna/setup_xclbin_firmware.sh exists:
-  if [ ! -f "$SIGNER" ]; then
+  echo $DO_SIGNING
+  if [ ! -f "$SIGNER" ] || [ $DO_SIGNING -eq 0 ]; then
```

Comment:
Please change this to

 if [ $DO_SIGNING -eq 0 ]
 echo "**** Skipping XCLBIN signing: DO_SIGNING set to 0 ****"
 elif [ ! -f "$SIGNER" ]
    echo "**** Skipping XCLBIN signing: $SIGNER not found ****"
 else
   ...
   
so that the message remains accurate.